components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/StatUpdateStorePublisherDomain.java

```diff
@@ -0,0 +1,57 @@
+/*
+*  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package org.wso2.carbon.apimgt.impl.utils;
+
+import org.apache.axis2.clustering.ClusteringAgent;
+import org.apache.axis2.clustering.ClusteringFault;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.core.AbstractAdmin;
+
+import java.io.Serializable;
+
+/**
+ * This class contains the functions to update stat publishing in Store-Publisher domain
+ */
+public class StatUpdateStorePublisherDomain extends AbstractAdmin implements Serializable {
```

Comment:
No need to make this Serializable. And this doesn't need to be a AdminService. Since what this does is creating a cluster message and send it, same thing could have been done from StatsUpdateUtil, isn't it?